packages/fdc3-get-agent/src/ui/NullIntentResolver.ts

```diff
@@ -0,0 +1,16 @@
+import { AppIntent } from "@kite9/fdc3-standard";
+import { IntentResolver, IntentResolutionChoice } from "@kite9/fdc3-standard";
+import { Context } from "@kite9/fdc3-context";
+
+
```

Comment:
Can you add a comment about what this file is doing, why it exists? Is it used for testing, for development, is it publicly facing?